src/main/ts/Part2Ex3Optional.ts

```diff
@@ -107,8 +116,10 @@ You can do this with fold, but getOr is a shortcut.
 */
 
 // TODO: Using getOr, take an Optional<{age: number}> and turn it into an {age: number}, using a default value of 0.
+export const toValueOr = (input: Optional<{age: number}>): {age: number} => input.getOr({age: 0});
 
 // TODO: Write the same function using fold
+export const toValueOrWithFold = (input: Optional<{age: number}>): {age: number} => input.fold(() => {return {age: 0};}, (input: {age: number}) => input );
```

Comment:
```suggestion
export const toValueOrWithFold = (input: Optional<{age: number}>): {age: number} => input.fold(() => ({age: 0}), (input: {age: number}) => input);
```
As you're using newer ECMAScript syntax here you don't need the `return` statement.